diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/StringConcatenationRule.kt

```diff
@@ -71,4 +88,63 @@ class StringConcatenationRule : Rule("string-concatenation") {
         return operationReference
                 ?.getFirstChildWithType(PLUS) != null
     }
+
+    private fun fixBinaryExpressionWithConcatenation(node: ASTNode?): ASTNode? {
+        return node?.let {
+            val binaryExpressionNode = node.psi as KtBinaryExpression
```

Comment:
```suggestion
            val binaryExpressionPsi = node.psi as KtBinaryExpression
```
I suggest not to use 'node' in the name of variable which isn't `ASTNode`.